### PR TITLE
Fix a vulnerability for resource leak

### PR DESCRIPTION
**Pull request by yugeeklab**
Put the method "close()" into appropriate position.

Signed-off-by: yugeeklab <yugeeklab@gmail.com>

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by riyenas0925** [comment]
try-with-resources looks better than try-catch-finally.

### Reference
* https://rules.sonarsource.com/java/RSPEC-2093
* https://www.oracle.com/java/technologies/javase/seccodeguide.html

**Comment by yugeeklab** [comment]
Done. Use try_with_resources statement.

File: src/main/java/oss/fosslight/util/FileUtil.java [hunk 1]
```diff
@@ -189,24 +189,25 @@ public static String makeValidFileName(String fileName, String replaceStr) {
...
+			
+			// 압축을 할 대상이 file이면 zipFile 메소드를,
+			// 폴더이면 zipFolder 메소드를 호출한다.
+			if (inputFile.isFile()) {
+				zipFile(zipOutputStream, inputFile, "");
+			} else if (inputFile.isDirectory()) {
+				zipFolder(zipOutputStream, inputFile, "", zipRootEntryName);
+			}
+		} catch (Exception e) {
+			throw e;
```
Comment (soimkim) [inline src/main/java/oss/fosslight/util/FileUtil.java:209]: There is a part that is not wrapped with try catch in calling this function. (Ex- ./main/java/oss/fosslight/service/impl/VerificationServiceImpl.java)
Please check it out.

Comment (yugeeklab) [reply src/main/java/oss/fosslight/util/FileUtil.java:209]: I don't know what it is best practice but I see a hierarchy that 

getNoticeHtml try_catch(VerificationController) -> updateStatusWithConfirm throws(VerificationServiceImpl) -> zip throws

So In the controller, The error will be handled.

Comment (soimkim) [reply src/main/java/oss/fosslight/util/FileUtil.java:209]: Great.

**Pull request merged**